### DISCUSSION TITLE
Correlation Handling

### DISCUSSION THREAD

**Discussion by seemantsingh-code** [guillermo-navas-palencia/optbinning#288] [Q&A] [answered]
Does the library offer any methods or options to analyze or address variable correlation both pre and post binning? I want to drop highly correlated variables.

**Comment by guillermo-navas-palencia** [comment, marked as answer]
Hi @maxsanwal.

The OptBinning library does not offer such methods. For analyzing correlation post-binning, you could employ rank correlation methods or simply Pearson's correlation. These methods also apply to pre-binning numerical variables. However, for categorical variables, other methods are required.
